src/java/org/apache/cassandra/index/sai/utils/ScoredPrimaryKeyIterator.java

```diff
@@ -24,9 +24,9 @@
  * An iterator over {@link ScoredPrimaryKey} ordered by score in descending order. Ascending order might be added
  * later.
  */
-public abstract class OrderIterator extends AbstractIterator<ScoredPrimaryKey> implements Closeable
+public abstract class ScoredPrimaryKeyIterator extends AbstractIterator<ScoredPrimaryKey> implements Closeable
```

Comment:
I think we ended up not needing any extra fields or methods here, let's just replace with CloseableIterator<ScoredPrimaryKey> (now that we're adding an emptyIterator method to CI)